model/version.go

```diff
@@ -745,9 +745,10 @@ func constructManifest(v *Version, projectRef *ProjectRef, moduleList ModuleList
 		}
 	}
 
+	isMergePatch := v.Requester == evergreen.MergeTestRequester
```

Comment:
https://github.com/evergreen-ci/evergreen/blob/cad3a44bc6a2eff74ce9a742960f3bb8dd102962/globals.go#L1023

we can probably just use this function instead